### ISSUE TITLE
OLH enlarge figure popup close button not focusable

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5277]
see a11y audit #5148 

The enlarged figure popup on the OLH theme can only be dismissed by keyboard with the ESC key.

There is a close button, but this is not focusable / keyboard reachable.

It should be.

**Comment by mauromsl** [comment]
closed by #5548
